zoom:
  ros__parameters:
    name: zoom
    publish_rate: 30.0
    read_rate: 30.0
    flip: 0
    width: 640.0
    height: 360.0
    fourcc: [M, J, P, G] # 1196444237.0
    topic_name: /video_mapping/zoom/image_raw
    cam_info_topic: None
    cam_info_period: 5
    intrinsic: True
    type: USB
    video_path: None
    object_detection: False
    semantic_segmentation: False
    qr_scan: False
    data_capture_csv: True
    data_capture_video: False
    # file_path: ''
    # frame_id: camera
    use_sim_time: False
    video_stream_recovery_time: 2
    video_stream_recovery_tries: 10

left:
  ros__parameters:
    name: left
    publish_rate: 30.0
    read_rate: 30.0
    flip: 0
    width: 640.0
    height: 360.0
    fourcc: [M, J, P, G] # 1196444237.0
    topic_name: /video_mapping/left/image_raw
    cam_info_topic: None
    cam_info_period: 5
    intrinsic: True
    type: USB
    video_path: None
    object_detection: False
    semantic_segmentation: False
    qr_scan: False
    data_capture_csv: True
    data_capture_video: False
    # file_path: ''
    # frame_id: camera
    use_sim_time: False
    video_stream_recovery_time: 2
    video_stream_recovery_tries: 10

right:
  ros__parameters:
    name: right
    publish_rate: 30.0
    read_rate: 30.0
    flip: 0
    width: 640.0
    height: 360.0
    fourcc: [M, J, P, G] # 1196444237.0
    topic_name: /video_mapping/right/image_raw
    cam_info_topic: None
    cam_info_period: 5
    intrinsic: True
    type: USB
    video_path: None
    object_detection: False
    semantic_segmentation: False
    qr_scan: False
    data_capture_csv: True
    data_capture_video: False
    # file_path: ''
    # frame_id: camera
    use_sim_time: False
    video_stream_recovery_time: 2
    video_stream_recovery_tries: 10

back:
  ros__parameters:
    name: back
    publish_rate: 30.0
    read_rate: 30.0
    flip: 0
    width: 640.0
    height: 360.0
    fourcc: [M, J, P, G] # 1196444237.0
    topic_name: /video_mapping/back/image_raw
    cam_info_topic: None
    cam_info_period: 5
    intrinsic: True
    type: USB
    video_path: None
    object_detection: False
    semantic_segmentation: False
    qr_scan: False
    data_capture_csv: True
    data_capture_video: False
    # file_path: ''
    # frame_id: camera
    use_sim_time: False
    video_stream_recovery_time: 2
    video_stream_recovery_tries: 10

inner:
  ros__parameters:
    name: inner
    publish_rate: 30.0
    read_rate: 30.0
    flip: 0
    width: 640.0
    height: 360.0
    fourcc: [M, J, P, G] # 1196444237.0
    topic_name: /video_mapping/inner/image_raw
    cam_info_topic: None
    cam_info_period: 5
    intrinsic: True
    type: USB
    video_path: None
    object_detection: False
    semantic_segmentation: False
    qr_scan: False
    data_capture_csv: True
    data_capture_video: False
    # file_path: ''
    # frame_id: camera
    use_sim_time: False
    video_stream_recovery_time: 2
    video_stream_recovery_tries: 10
